static/js/stream_muting.js

```diff
@@ -67,4 +68,24 @@ export function update_is_muted(sub, value) {
         )}'] #sub_is_muted_setting .sub_setting_control`,
     );
     is_muted_checkbox.prop("checked", value);
+
+    $(".mute-note").toggleClass("hide-mute-note", !value);
```

Comment:
These selectors are all dangerously imprecise; when you do an unscoped selector like this, it applies to all `mute-note` objects in the app, not just the ones you want to change.  What you want is to find these elements within the `.subscription_settings[data-stream-id='${CSS.escape(...` scope, so we only update the row we want to.